packages/react-storage/src/components/StorageBrowser/actions/configs/types.ts

```diff
@@ -117,6 +117,9 @@ export type CustomActionConfigs = Record<
   ActionViewConfig | ActionHandler
 >;
 
+/**
+ * @internal
+ **/
```

Comment:
Comment syntax s slightly off and let's go ahead and mark these as `@unstable` as well:
```suggestion
/**
 * @internal @unstable
 */
```